src/quicr_client_raw_session.h

```diff
@@ -0,0 +1,251 @@
+/*
+ *  quicr_client_raw_session.h
+ *
+ *  Copyright (C) 2023
+ *  Cisco Systems, Inc.
+ *  All Rights Reserved
+ *
+ *  Description:
+ *      This file defines a session layer between the client APIs and the
+ *      transport that uses raw data packets (either UDP or QUIC).
```

Comment:
please add a note to clarify what does raw data packets mean for UDP or QUIC in the context of session

Reply:
```cpp
 *      transport that uses raw data packets, namely UDP or QUIC.
```

I'll change it to that.  Raw just means UDP or QUIC.  The other one coming is quicr_client_h3_session.h.  It will make more sense with that present.

Reply:
yes, i do understand the context. My comment for someone external who look at the header, it is lacking context on being raw .. a note on the context will help